imhamt/src/content.rs

```diff
@@ -28,7 +28,7 @@ impl<K: Clone, V> KV<K, V> {
 pub enum SmallVec<T> {
     One(T),
     //Two(T, T),
-    Many(Vec<T>),
+    Many(Box<[T]>),
 }
```

Comment:
You could soon (now?) be able try to use const generics for the inline variant.

Reply:
Is the branching between two variants worth having an inline form, BTW?
I had a similar [discussion](https://github.com/tokio-rs/bytes/issues/268) about `Bytes` recently, and they decided to cut down on internal dynamic polymorphism to speed up the main use cases.

Reply:
Here the main use case is often one output to track.